### DISCUSSION TITLE
How to expand all categary by config in storybook V7.

### DISCUSSION THREAD

**Discussion by Flcwl** [storybookjs/storybook#23194] [Help]
I want to expand sider menus by default when open the storybook page. like it below.

![image](https://github.com/storybookjs/storybook/assets/26050644/6b59d3fc-4efd-4acf-b8f7-6bd651bb00d5)

**Comment by vvs09** [comment]
I don't think the sidebar's expand/collapse state is not customizable out of the box. but there is a work around to it. Try

// Create a new decorator in your preview.js
export const parameters = {
  decorators: [
    (Story, context) => {
      useEffect(() => {
        addons.getChannel().emit('expandAll');
      }, []);

      return <Story {...context} />
    }
  ]
}

use this decorator for your components

storiesOf('Component', module)
  .addDecorator(withDecorator)
  .add('Story', () => <Component />);
